CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Reduce workspace size from 
```

Comment:
From?

Reply:
I updated the changelist with the numbers. Gains are minimal :(